app/src/shared/components/TextField/NumberField/index.jsx

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import React, { useCallback, useRef } from 'react'
+import cx from 'classnames'
+
+import SvgIcon from '$shared/components/SvgIcon'
+
+import styles from '../textField.pcss'
+
+type Props = {
+    className?: string,
+    onAutoComplete?: (boolean) => void,
+    type?: string,
+    min?: number,
+    max?: number,
+    step?: number,
+    hideButtons?: boolean,
+}
+
+const NumberField = ({ className, onAutoComplete, hideButtons, ...props }: Props) => {
+    const inputRef = useRef(null)
+
+    const onAnimationStart = useCallback(({ animationName }: SyntheticAnimationEvent<EventTarget>) => {
```

Comment:
Don't we pass `onAnimationStart` to this component in `TextField`? It would make auto-complete detection redundant here.